avogadro/qtgui/hydrogentools.cpp

```diff
@@ -123,6 +123,10 @@ void HydrogenTools::adjustHydrogens(RWMolecule& molecule, Adjustment adjustment)
            it != itEnd; ++it) {
         RWAtom newH(molecule.addAtom(1));
         newH.setPosition3d(*it);
+
+        const unsigned char* color = Core::Elements::color(1);
```

Comment:
When we start using the default atomic colors (instead of black) for when the color isn't set, we probably won't need this section and a few other sections.